mindjunkies/forums/tests.py

```
@@ -0,0 +1,88 @@
+        self.client.login(username="testuser", password="testpass")
+        response = self.client.post(
+            reverse("comment_submission", kwargs={"course_slug": self.course.slug, "topic_slug": self.forum_topic.slug}),
+            {"content": "New Comment"}
+        )
+        self.assertEqual(response.status_code, 302)  # Redirect on success
+        self.assertTrue(ForumComment.objects.filter(content="New Comment").exists())
+
+    def test_reply_submission_view(self):
+        self.client.login(username="testuser", password="testpass")
+        response = self.client.post(
+            reverse("reply_submission", kwargs={"course_slug": self.course.slug, "topic_slug": self.forum_topic.slug, "comment_id": self.comment.id}),
```

Ruff (0.8.2)

Line too long (150 > 119)

(E501)